sandman/source/input.cpp

```diff
@@ -194,14 +193,14 @@ void Input::Process()
 	
 	// Read up to 64 input events at a time.
 	static unsigned int const s_EventsToReadCount = 64;
-	input_event l_Events[s_EventsToReadCount];
-	static auto constexpr s_EventSize = sizeof(input_event);
-	static auto const s_EventBufferSize = s_EventsToReadCount * s_EventSize;
+	input_event events[s_EventsToReadCount];
+	static constexpr std::size_t kEventSize{sizeof(input_event)};
```

Comment:
Yes, `kEventSize` could be declared with `auto` because the `sizeof` operator results in a constant expression of type `std::size_t` which matches the type of the `count` argument of the `read` function from `<unistd.h>`.